packages/ui/src/common/components/icons/BinIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { Icon } from './Icon'
+
+export const BinIcon = React.memo(({ className, onClick }: { className?: string; onClick?: () => void }) => (
+  <Icon
+    size="20"
+    viewBox="0 0 24 24"
+    preserveAspectRatio="xMidYMid meet"
+    fill="none"
+    color="currentColor"
+    className={className}
+    onClick={onClick}
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
A bunch of icons have this small problems already (and I keep on forgetting too) but the path attributes should be camel case too. Otherwise reacts just adds a warning in the console

Reply:
I fixed it for all icons with incorrect fill-rule/clip-rule